src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
You can get rid of the sequence number and use `ol.getUid()` instead. Maybe add a comment there that the generated uids are in ascending order. In the `ol.Tile` constructor, simply call `ol.getUid(this);`.